src/input-group/input-group.style.tsx

```diff
@@ -80,40 +82,19 @@ export const MainInput = styled(Input)<MainInputStyleProps>`
     // overwrite default styles
     background: transparent;
     border: none;
+    padding: 0;
```

Comment:
noticed that input color takes the browser default `#000`

because of that, for readonly, there's a slight mismatch with the addon text which is N1

Reply:
Good catch. Let me go rectify